crates/router/src/connector/iatapay/transformers.rs

```diff
@@ -515,7 +515,7 @@ pub struct IatapayErrorResponse {
 #[derive(Deserialize, Debug, Serialize)]
 pub struct IatapayAccessTokenErrorResponse {
     pub error: String,
-    pub path: String,
+    pub path: Option<String>,
```

Comment:
Is this an optional field from connector?

Reply:
When i try to pass invalid client keys , It started throwing 5xx because of deserilization error in `IatapayAccessTokenErrorResponse`. So i made the path field as optional. 
<img width="1017" alt="Screenshot 2024-04-04 at 1 12 34 AM" src="https://github.com/juspay/hyperswitch/assets/121822803/d0bc4162-ffac-4749-8b0c-de223234f1a7">